api/routes/ws.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:dart_frog/dart_frog.dart';
+import 'package:dart_frog_web_socket/dart_frog_web_socket.dart';
+
+import '_middleware.dart';
+
+/// Websocket Handler
+Future<Response> onRequest(RequestContext context) async {
+  final handler = webSocketHandler((channel, protocol) {
+    channel.stream.listen((message) {
+      // Global pointer definition
+      pointerOffsets.add(message.toString());
+      final offsets = context.read<List<dynamic>>();
+
+      // Send back offsets to clients
+      channel.sink.add(offsets.toString());
+    });
+  });
+  return handler(context);
```

Comment:
We'll also have to broadcast this using broadcast_bloc. Good attempt!

Reply:
Yes I think this is the tutorial for this : https://dartfrog.vgv.dev/docs/tutorials/web_socket_counter